package/raphael.js

```diff
@@ -5875,7 +5905,7 @@
                 }
             }
             timestamp = params.start || anim.start || +new Date;
-            e = {
+            element.e =  e = {
```

Comment:
Remember that this object needs to be deleted to avoid memory leak and that if there were two simultaneous animations, it MAY be a problem.